flink-libraries/flink-table/src/main/scala/org/apache/flink/table/sources/TableSource.scala

```diff
@@ -60,4 +60,14 @@ trait TableSource[T] {
     */
   def explainSource(): String = ""
 
+  /**
+    * Gets the name which is used by the visualization and logging during runtime.
+    *
+    * @return Name of the [[TableSource]].
+    */
+  def getRuntimeName(): String = {
```

Comment:
I would like to avoid changing the `TableSource` interface again. I think we can use `explainSource` instead.

Reply:
Hi @fhueske, thanks for the suggestion. I'll remove this method and add a default implementation for `explainSource()`.

Reply:
Shall I reserve the existing `explainSource()`  implementations or use a unified one?

Reply:
I would keep the existing implementations.

Reply:
Fine. I just added a default implementation in `TableSource` trait and kept the existing ones in subclasses.